packages/tx/src/baseTransaction.ts

```diff
@@ -83,7 +83,7 @@ export abstract class BaseTransaction<TransactionObject> {
   protected DEFAULT_HARDFORK: string | Hardfork = Hardfork.Istanbul
 
   constructor(txData: TxData | AccessListEIP2930TxData | FeeMarketEIP1559TxData) {
-    const { nonce, gasLimit, to, value, data, v, r, s, type } = txData
+    const { data, nonce, gasLimit, r, s, to, type, v, value } = txData
```

Comment:
hm why reorder these? better to just leave them as is?

Reply:
My guess is that this is the linter 😄

Reply:
Ah yeah I have a reordering thing setup, can undo